### PR TITLE
feat (executor): new ssh executor

### LINKED ISSUE #14: feat (executor): remote exec
via ssh
same as exec with

input:
 - privatekey
 - username (default is os.username)
 - password (mandatory if privatekey is empty)
 - hostname (mandatory
 - port (default 22)

### PR DESCRIPTION
**Pull request by fsamin**
close #14 

Signed-off-by: François SAMIN <francois.samin@gmail.com>

### CODE REVIEW

File: executors/ssh/executor.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+func privateKey(file string) (key ssh.Signer, err error) {
+	//Default private key is $HOME/.ssh/id_rsa
+	if file == "" {
+		usr, err := user.Current()
+		if err != nil {
+			return nil, err
+		}
+		file = usr.HomeDir + "/.ssh/id_rsa"
+	}
+	fmt.Println(file)
```
Comment (sguiheux) [inline executors/ssh/executor.go]: remove fmt

File: executors/ssh/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - result.code ShouldEqual 0
+    - result.timeseconds ShouldBeLessThan 1
+
+```
+
+## Output
+
+```yaml
+executor
+stdout
```
Comment (yesnault) [inline executors/ssh/README.md]: systemout

File: executors/ssh/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+stderr
```
Comment (yesnault) [inline executors/ssh/README.md]: systemerr

File: executors/ssh/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+err
+code
+timeseconds
+timehuman
+```
+
+- result.timeseconds & result.timehuman: time of execution
+- result.executor.executor.script: script executed
+- result.err: if exists, this field contains error
+- result.stdout: Standard Output of executed script
```
Comment (yesnault) [inline executors/ssh/README.md]: systemout & systemerr

File: executors/ssh/executor.go [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/fsamin/go-dump"
+	"github.com/mitchellh/mapstructure"
+	"golang.org/x/crypto/ssh"
+
+	"github.com/runabove/venom"
+)
+
+// Name for test exec
```
Comment (yesnault) [inline executors/ssh/executor.go]: Name for test ssh

**Comment by yesnault** [comment]
Lgtm

**Pull request merged**